### ISSUE TITLE
priorities 3.44

### ISSUE THREAD

**Issue by smanders** [smanders/externpro#113]
### 16.10.1 ###
- [x] [update the cmake version](https://github.com/smanders/externpro/issues/107)
- [x] [update the clang-format version](https://github.com/smanders/externpro/issues/102)
- [x] [internal: mkpatch step fails with newer version of cmake](https://github.com/smanders/externpro/issues/110)
- [x] [internal: stage directory markings optional](https://github.com/smanders/externpro/issues/111)
- [x] [build clang-format Visual Studio extension](https://github.com/smanders/externpro/issues/112)

### 16.10.2 ###
- [x] [option to build against gtk3](https://github.com/smanders/externpro/issues/103)
- [x] [fix xpSourceListAppend](https://github.com/smanders/externpro/issues/114)
- [x] [update the rapidjson version](https://github.com/smanders/externpro/issues/34)
- [x] [add node.js version 6 as an option](https://github.com/smanders/externpro/issues/91)
- [x] [enhance (or remove) flatnode cmake script](https://github.com/smanders/externpro/issues/80)
- [x] [patch project updates](https://github.com/smanders/externpro/issues/116)
- [x] [default BUILD|INSTALL_COMMAND not working](https://github.com/smanders/externpro/issues/117)
- [x] [unix build speed](https://github.com/smanders/externpro/issues/118)
- [x] [webpro support](https://github.com/smanders/externpro/issues/115)
- [x] [CMakeParseArguments is no longer a cmake module](https://github.com/smanders/externpro/issues/119)

### 16.10.3 ###
- [x] [rapidjson, libgit2, msvc inttypes.h mess](https://github.com/smanders/externpro/issues/120)
- [x] webpro should build against current externpro
- [x] vantage should build against current externpro
- [x] internpro should build against current externpro

### current ###
- [x] [create header resources via wxInclude](https://github.com/smanders/externpro/issues/123)
- [x] [add propgrid to wx_libs](https://github.com/smanders/externpro/issues/121)
- [x] [remove node.js version 5](https://github.com/smanders/externpro/issues/122)
- [x] [update zlib and gsoap download URLs](https://github.com/smanders/externpro/issues/125)
- [x] [xpSourceListAppend enhancements](https://github.com/smanders/externpro/issues/124)
- [x] [minor find script cleanup](https://github.com/smanders/externpro/issues/126)
- [x] [allow projects that use externpro to set CPACK_PACKAGE_VENDOR](https://github.com/smanders/externpro/issues/127)
- [x] [update boost](https://github.com/smanders/externpro/issues/39)
- [x] [update the cmake version](https://github.com/smanders/externpro/issues/129)

### next ###
- [ ] debug symbols
- [ ] [OpenCV](http://opencv.org/) (Blake)
- [ ] [wx build should avoid dependencies on system installed libraries](https://github.com/smanders/externpro/issues/104)
- [ ] [update protobuf](https://github.com/smanders/externpro/issues/75)
- [ ] [add FFmpeg](https://github.com/smanders/externpro/issues/17)
- [ ] [update geotrans](https://github.com/smanders/externpro/issues/23)
- [ ] [libRaptorQ shared libs](https://github.com/smanders/externpro/issues/71)
- [ ] [update glew](https://github.com/smanders/externpro/issues/18)
- [ ] [add intel TBB](https://github.com/smanders/externpro/issues/73)
- [ ] [openssl link errors](https://github.com/smanders/externpro/issues/48)

### 16.11.1 (16.08.2) ###
- [x] build against latest externpro
- [x] update Sdvideo
- [x] palam updates (version 1.1.1.0)
- [x] add NG_GDP
- [x] add multiple version build support to NG_GDP
- [x] NG_GDP sdl_ggdp-targets.cmake INTERFACE_LINK_LIBRARIES

### 16.11.2 ###
- [x] palam built against nodev6 (version 1.1.2.0)

### 16.11.3 (16.08.3) ###
- [x] update SkyLynx firmware 16.11.18, vxWorks 1.15.0.6
- [x] update NG_GDP 16.02

### 17.02.1 (16.08.4) ###
- [x] update SkyLynx firmware 17.01.26, vxWorks 1.15.0.8
- [x] update GDP 17.00
- [x] update palam (version 1.1.3.0)

### 17.03.1 (16.08.5) ###
- [x] update SkyLynx firmware 17.03.09, vxWorks 1.16.0.0
- [x] update GDP 17.01
- [x] update GDP 17.02

### 17.03.2 (16.08.6) ###
- [x] update GDP 17.03

## non-externpro (Vantage, system) ##
- [ ] [devtoolset-4](https://www.softwarecollections.org/en/scls/rhscl/devtoolset-4/) aka gcc 5.2.1 (ticket 8407)
- [x] cmake cleanup (ticket 8419)
  - [x] cleanup library dependency graph (ExeLib complete, TestPlugin waiting on plugin reorg)
  - [x] javascript protobuf custom target shouldn't be run every time
  - [x] clang-format version 3.9.0 applied to files that slipped through
  - [x] Sdmsg link warnings on Windows
  - [x] `xpEnableWarnings()` the default
  - [x] fix some gcc compiler warnings
  - [x] add .proto files to .gitattributes
  - [x] cmake/repo sync-up
  - [x] remove SunOS (Solaris) specific things no longer needed
- [ ] turn clang-format SortIncludes on

**Comment by smanders** [comment]
3.44 development is complete
